src/User32/User32.cs

```diff
@@ -246,6 +246,19 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern int GetWindowThreadProcessId(IntPtr hWnd, out int lpdwProcessId);
 
+        /// <summary>
+        /// Attaches or detaches the input processing mechanism of one thread to that of another thread.
+        /// </summary>
+        /// <param name="idAttach">The identifier of the thread to be attached to another thread.The thread to be attached cannot be a system thread.</param>
+        /// <param name="idAttachTo">The identifier of the thread to which idAttach will be attached.This thread cannot be a system thread. A thread cannot attach to itself. Therefore, idAttachTo cannot equal idAttach.</param>
+        /// <param name="fAttach">If this parameter is TRUE, the two threads are attached.If the parameter is FALSE, the threads are detached.</param>
+        /// <returns>If the function succeeds, the return value is nonzero. If the function fails, the return value is zero.To get extended error information, call GetLastError. Windows Server2003 and WindowsXP:  There is no extended error information; do not call GetLastError.This behavior changed as of WindowsVista.</returns>
+        // [DllImport(nameof(User32), SetLastError = true)]
+        // public static extern bool AttachThreadInput(int idAttach, int idAttachTo, bool fAttach);
+        [DllImport("user32.dll", EntryPoint = "AttachThreadInput")]
```

Comment:
`"user32.dll"` should be `nameof(User32)`, no need to specify the `EntryPoint` and the function use `SetLastError` for it's error signaling so it should have `, SetLastError = true`.